app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
```

Comment:
Please remove the commented out code. As a rule of thumb we do not commit commented out lines, with small exceptions like initializers / config/env files